package.json

```diff
@@ -46,72 +46,72 @@
     "@angular/platform-browser-dynamic": "15.1.0",
     "@angular/platform-server": "15.1.0",
     "@angular/router": "15.1.0",
-    "@nrwl/angular": "15.4.5",
-    "marked": "^4.2.4",
+    "@nrwl/angular": "15.6.2",
+    "marked": "^4.2.12",
     "front-matter": "^4.0.2",
-    "rxjs": "7.5.7",
-    "tslib": "^2.4.0",
-    "zone.js": "~0.11.8"
+    "rxjs": "7.8.0",
```

Comment:
In general, I think its safer to only update the packages Nx updates. It also helps keep compatibility with Angular release versions. Revert the `rxjs, tslib, and zone.js` changes